storm-core/src/jvm/backtype/storm/utils/MutableObject.java

```diff
@@ -18,7 +18,7 @@
 package backtype.storm.utils;
 
 public class MutableObject {
```

Comment:
I can probably remove the changes I made to the class but it is used in other places in the code still.
```
$ find ./storm-core/ -iname \*.clj | xargs grep -l MutableObject
./storm-core//src/clj/backtype/storm/daemon/acker.clj
./storm-core//src/clj/backtype/storm/daemon/executor.clj
./storm-core//src/clj/backtype/storm/messaging/loader.clj
./storm-core//src/clj/backtype/storm/util.clj
```